chicago/bills.py

```diff
@@ -160,40 +86,144 @@ def scrape(self):
+                                    organization=action['Action\xa0By'],
+                                    classification=ACTION_CLASSIFICATION[action_description])
+                    if 'url' in action['Action\xa0Details'] :
+                        action_detail_url = action['Action\xa0Details']['url']
+                        result, votes = self.extractVotes(action_detail_url)
+
+                        if votes :
+                            action_vote = Vote(legislative_session=self.session, 
+                                               motion_text=action_description,
+                                               classification='bill-passage',
+                                               start_date=action_date,
+                                               result=result,
+                                               bill=bill.identifier)
+                            action_vote.add_source(action_detail_url)
+                            for option, voter in votes :
+                                action_vote.vote(option, voter)
+                        
+                            all_votes.append(action_vote)
+
+            except KeyError :
+                if action_description not in ('Direct Introduction',
+                                              'Remove Co-Sponsor(s)',
+                                              'Held in Committee',
+                                              'Deferred and Published') :
+                    print(action_description)
+                    raise
+
+
+        return all_votes
+
+
+    def addDetails(self, bill, detail_url) :
+        detail_page = self.lxmlize(detail_url)
+        detail_div = detail_page.xpath(".//div[@id='ctl00_ContentPlaceHolder1_pageDetails']")[0]
+
+        legislation_details = self.parseDetails(detail_div)
+        
+
+        for related_bill in legislation_details.get('Related files', []) :
+            # need different relation_type
+            bill.add_related_bill(identifier = related_bill['label'],
+                                  legislative_session = self.session,
+                                  relation_type='replaces')
+
+        for i, sponsor in enumerate(legislation_details.get('Sponsors', [])) :
+            if i == 0 :
+                primary = True
+                sponsorship_type = "Primary"
+            else :
+                primary = False
+                sponsorship_type = "Regular"
+
+            bill.add_sponsorship(sponsor['label'], sponsorship_type,
+                                 'person', primary)
+
+        if u'Topics' in legislation_details :
+            for subjuct in legislation_details[u'Topics'].split(',') :
+                bill.add_subject(subject)
+
+        for attachment in legislation_details.get(u'Attachments', []) :
+            bill.add_version_link(attachment['label'],
+                                  attachment['url'],
+                                  media_type="application/pdf")
```

Comment:
Legistar sometimes makes available an OCRed version of the pdfs of the bills. These are not an 'available form' of the bills. However, they are very useful to folks who may want to be working with this data down the line? How should we attach this? Would this be a "document"?